src/include/job.h

```diff
@@ -847,6 +846,19 @@ task_find	(job		*pjob,
 #define JOB_STATE_FINISHED	9
 
 
+#define JOB_STATE_LTR_BEGUN 'B'
+#define JOB_STATE_LTR_EXITING 'E'
+#define JOB_STATE_LTR_FINISHED 'F'
+#define JOB_STATE_LTR_HELD 'H'
+#define JOB_STATE_LTR_MOVED 'M'
+#define JOB_STATE_LTR_QUEUED 'Q'
+#define JOB_STATE_LTR_RUNNING 'R'
+#define JOB_STATE_LTR_SUSPENDED 'S'
+#define JOB_STATE_LTR_TRANSIT 'T'
+#define JOB_STATE_LTR_USUSPENDED 'U'
```

Comment:
I actually am not familiar with this state. I chose "user suspended" because that's how we describe the U state in the ref guide. Would you still like me to change it to user busy?

Reply:
We do consider this as userbusy in the scheduler (set_job_state function) as well. So I guess it will be better to be consistent.

Reply:
When we designed the state, we called it user  busy.  I don't like User suspended because it could mean multiple things.  It could mean the user did a qsig -s suspend or cycle harvesting.  Userbusy is more explicit to cycle harvesting.